teachertool/src/components/MainPanel.tsx

```diff
@@ -3,18 +3,31 @@ import css from "./styling/MainPanel.module.scss";
 import { SplitPane } from "./SplitPane";
 import { RubricWorkspace } from "./RubricWorkspace";
 import { ProjectWorkspace } from "./ProjectWorkspace";
+import { getLastSplitPosition, setLastSplitPosition } from "../services/storageService";
 
 interface IProps {}
 
 export const MainPanel: React.FC<IProps> = () => {
+    const defaultSize = "50%";
```

Comment:
Do we want to keep this at 50%? I know in the designs we have the split with the rubric side of things wider than the iframe. Wondering if we want to have that wider design be the default.

Reply:
I don't feel strongly, but I think 50% is a reasonable starting point and we can observe if/how teachers tend to resize it when they use it. If there is consistent expansion in one direction, we can make that the default.

Reply:
Makes sense. I just wanted to bring it up because I think on most monitors, having the splitter at 50% would show the large simulator instead of the small simulator in the corner and I wasn't sure if we were okay with that.

Reply:
Yeah, that's certainly the case on my screen, but let's see how it plays out.